modules/ocf_hpc/manifests/miniconda/apt.pp

```diff
@@ -0,0 +1,16 @@
+# Include miniconda apt repo
```

Comment:
I would move this file into conda.pp

I also think it probably is better to name it conda.pp since that is the package name.